pages/clustering/high-availability.mdx

```diff
@@ -36,11 +36,11 @@ One coordinator instance is the leader whose job is always to ensure one writeab
 own Raft log. Operations saved into the Raft log are those that are related to cluster management. Memgraph doesn't have its implementation of the Raft protocol. 
 For this task, Memgraph uses an industry-proven library [NuRaft](https://github.com/eBay/NuRaft).
 
-You can start the coordinator instance by specifying `--coordinator-id` and 
-`--coordinator-port` flags. The coordinator instance only responds to
+You can start the coordinator instance by specifying `--coordinator-id`, 
+`--coordinator-port` and `--management-port` flags. The `--management-port` is used on follower coordinators to get health state of cluster from leader coordinator. The coordinator instance only responds to
```

Comment:
What happens when a leader fails and is removed from the cluster, recovers, and then attempts to rejoin the cluster as a follower? The way this is written now, it will fail because it doesn't know the management port. Is that accurate?

Reply:
@kgolubic that is not fully correct, it is not removed from the cluster, and it will not ping the leader to get a management port.

Reply:
I changed the wording a bit, let me know if you have additional comments